.buildkite/commands/test-android.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -eu
+
+MODE="full"
+INPUT="${1-}"
+while [ "$INPUT" != "" ]; do
+    case $INPUT in
+        --canary )
+            MODE="canary"
+            ;;
+        * )
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+    INPUT="${1-}"
+done
+
+echo '--- :node: Setup Node depenendencies'
```

Comment:
Merely grammatical suggestion, feel free to disregard. This could be applied to `test-ios.sh` too, if desired. 
```suggestion
echo '--- :node: Set up Node depenendencies'
```

Reply:
I appreciated this grammatical suggestion. Keep them coming!